tasks/main.yml

```diff
@@ -3,6 +3,9 @@
   apt: pkg=ntp={{ ntp_version }} state=present
 
 - name: Configure NTP
-  template: src=ntp.conf.j2 dest=/etc/ntp.conf
+  template: src={{ item }} dest=/etc/ntp.conf
+  with_first_found:
+    - "ntp.{{ ansible_distribution_release }}.conf.j2"
```

Comment:
Addressed in [1fe38cc](https://github.com/azavea/ansible-ntp/pull/2/commits/1fe38ccaf01eaefa4e2d83ceb4968232b36bbfcd).